db/fns.md

```diff
@@ -1658,18 +1664,30 @@ where the page of results should begin, and must all be specified if any
 are specified.  Typically these values would be drawn from the last item
 in the previous page.
 
-### get_task_group
+### get_task_group_project_ids
```

Comment:
should this be `get_sealed_task_group_project_ids` ? i feel like it would be safer to only allow it for sealed task groups.

Reply:
Problem here is that we only need this to check permissions :) To know if someone should be able to perform this action. 

We could as well drop those project scopes and ignore this for sealing

Reply:
It is a tough one. Maybe we should also have queue.sealTaskGroupForProject as a separate endpoint, that only does it on behalf of a single project that is specified by the caller. The problem with that though is then it doesn't make sense for a task group to be sealed or not, because it then becomes a property of taskgroup/project combination. I'm starting to think the idea of multiple projects per taskgroup might be too compilcated, and perhaps one taskgroup, one project, is going to make life a lot simpler and also still be pretty usable.